app/javascript/modules/listingDetailsAside/ListingDetailsAside.tsx

```diff
@@ -30,7 +29,7 @@ export const ListingDetailsAside = ({ listing, imageSrc }: ListingDetailsSidebar
           <ListingDetailsApplicationDate isApplicationOpen={isApplicationOpen} listing={listing} />
           <ListingDetailsLottery listing={listing} />
           {isApplicationOpen && <ListingDetailsInfoSession listing={listing} />}
-          <ListingDetailsApply listing={listing} />
+          {/* DAH-1136 <ListingDetailsApply listing={listing} /> */}
```

Comment:
we were investigating [expiring TODOs](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/expiring-todo-comments.md) recently — I wonder if it'd be possible to do something similar with JIRA tickets 🤔

Reply:
Luckily this jira is in the sprint so it should get resolved soon, but that's an excellent idea. Perhaps add the jira with an expiring TODO